### ISSUE TITLE
Hardcode-Tray does not change icons and makes franz icon invisible 

### ISSUE THREAD

**Issue by crneus** [bilelmoussaoui/Hardcode-Tray#564]
### Specifications
- Hardcode-Tray version : `4.2`
- Distribution : `Solus 3.9999`
- Gtk version : `3.22`
- Inkscape version : `0.92.4` 
- Cairo version : `` 
- Sni-qt version: ``
- Application name :  ``
- Application version : ` `

### Expected behavior and actual behavior
Should change the system tray icons to papirus icon theme
Does not work, hides franz icon.

### Steps to reproduce the problem
Running hardcode-tray use "apply"

Maybe this is because I installed the 4.2 version from the solus repo instead of 4.3 which currently not
available through the repo. But I dont know.
![screenshot from 2019-01-24 19-00-37](https://user-images.githubusercontent.com/46002776/51698414-66ebc500-200a-11e9-9614-46b956f9581d.png)

**Comment by crneus** [comment]
Okay strangly after I booted up my computer today franz appeard but with the papirus icon, not the default one. Even tho I deleted hardcode-tray yesterday. 

For testing reasons I installed hardcode-tray on my laptop today and ran the script. The same problem like I mentioned in my first comment occurs. 

![screenshot from 2019-01-25 16-23-04](https://user-images.githubusercontent.com/46002776/51754795-85f75f00-20bd-11e9-9629-e1dc7d9f2180.png)
